src/EnumServiceProvider.php

```diff
@@ -96,9 +96,9 @@ private function bootDoctrineType()
     private function bootValidationTranslation()
     {
         $this->publishes([
-            __DIR__.'/../resources/lang' => resource_path('lang/vendor/laravelEnum'),
+            __DIR__ . '/../lang' => lang_path('vendor/laravelEnum'),
```

Comment:
```suggestion
            app()->langPath() => lang_path('vendor/laravelEnum'),
```

Sorry, wasn't really clear before. I _think_ it should be like this.

Reply:
No, the first path is relative to this file in this project.